src/react-apps/applications/shared/src/components/AltinnCheckBox.tsx

```diff
@@ -44,6 +47,7 @@ export const AltinnCheckBox = (props: IAltinnCheckBoxComponentProvidedProps) =>
       disabled={props.disabled ? props.disabled : false}
       onKeyPress={props.onKeyPressFunction}
       tabIndex={0}
+      disableRipple={false}
```

Comment:
I think the designers have mentioned that the ripple effect is unwanted. Might have to check with them...